frontend/src/database/chat-db.ts

```diff
@@ -14,31 +14,33 @@ import type { Table } from 'dexie';
 
 export interface ChatMessage {
   id: string;
+  agentId: string;
   content: string;
   sender: 'user' | 'system';
   timestamp: Date;
+  failure: boolean;
 }
 
 class ChatDatabase extends Dexie {
   messages!: Table<ChatMessage, string>;
 
   constructor() {
     super('ChatDatabase');
-    this.version(1).stores({
-      messages: 'id, sender, timestamp',
```

Comment:
Does the previous version need to be kept as well or is it ok to just run the last migration?

Reply:
I'dI think both are ok, I don't think we will modify db often though. We can set it to 1 again if you want

Reply:
All good, just not sure about the convention